PWGLF/Tasks/Resonances/higherMassResonances.cxx

```diff
@@ -170,6 +170,11 @@ struct HigherMassResonances {
     // ConfigurableAxis axisdEdx{"axisdEdx", {20000, 0.0f, 200.0f}, "dE/dx (a.u.)"};
     // ConfigurableAxis axisPtfordEbydx{"axisPtfordEbydx", {2000, 0, 20}, "pT (GeV/c)"};
     // ConfigurableAxis axisMultdist{"axisMultdist", {3500, 0, 70000}, "Multiplicity distribution"};
+
+    // fixed variables
+    float rapidityMotherData = 0.5;
+    std::array<int, 6> numbers = {0, 1, 2, 3, 4, 5};
+    double beamMomentum = std::sqrt(13600 * 13600 / 4 - o2::constants::physics::MassProton * o2::constants::physics::MassProton); // GeV
   } config;
```

Comment:
The value `13600` is a magic number which appears multiple times in the code. Define a constant `beamEnergy` for it.